src/lib/stellar/horizonQueries.js

```diff
@@ -38,7 +38,16 @@ export async function fetchAccount(publicKey) {
         } catch (err) {
             // @ts-ignore
             if (err.response?.status === 404) {
-                throw error(404, 'account not funded on network')
+                try {
+                    fundWithFriendbot(publicKey)
+
+                    let account = await server.accounts().accountId(publicKey).call()
+                    return account
+                } catch (error) {
```

Comment:
```suggestion
                } catch (err) {
```

This will be more consistent with the rest of the codebase